src/api/index.js

```diff
@@ -65,6 +65,9 @@ const apiAuthPost = (endpoint, body, authHeader) => {
 const apiAuthPut = (endpoint, body, authHeader) => {
   return axios.put(`${getApiUrl()}${endpoint}`, body, { headers: authHeader });
 };
+const apiAuthDelete = (endpoint, authHeader) => {
```

Comment:
apiAuth methods for all of the CRUD operations except for delete. I implemented that here.